pkgs/top-level/aliases.nix

```diff
@@ -836,6 +836,10 @@ mapAliases ({
   linux_rpi2 = linuxKernel.kernels.linux_rpi2;
   linux_rpi3 = linuxKernel.kernels.linux_rpi3;
   linux_rpi4 = linuxKernel.kernels.linux_rpi4;
+  linux_xanmod_stable = throw "linux_xanmod_stable was removed because it has reached its end of life upstream";
+  linuxPackages_xanmod_stable = throw "linuxPackages_xanmod_stable was removed because it has reached its end of life upstream";
+  linux_xanmod_tt = throw "linux_xanmod_tt was removed because upstream no longer offers this option"; # Added 2022-11-01
```

Comment:
Those are not the actial reasons if I'm not mistaken? They're just renamed, aren't they?

Reply:
Well, yes and no, no because it's renamed, and yes because the last version was 5.19 and that's EOL already.

Reply:
Well, the main version is currently the stable version and the latest version at the same time so it is a rename from stable -> main.
We could also keep both versions pointing to the main version in case upstream want's to introduce a "latest" version again.